src/nvim/window.c

```diff
@@ -488,6 +488,107 @@ do_window (
 }
 
 /*
+* Figure out the address type for ":wnncmd".
+*/
+void get_wincmd_addr_type(char_u *arg, exarg_T *eap)
+{
+  switch (*arg) {
+    case 'S':
+    case Ctrl_S:
+    case 's':
+    case Ctrl_N:
+    case 'n':
+    case 'j':
+    case Ctrl_J:
+    case 'k':
+    case Ctrl_K:
+    case 'T':
+    case Ctrl_R:
+    case 'r':
+    case 'R':
+    case 'K':
+    case 'J':
+    case '+':
+    case '-':
+    case Ctrl__:
+    case '_':
+    case '|':
+    case ']':
+    case Ctrl_RSB:
+    case 'g':
+    case Ctrl_G:
+    case Ctrl_V:
+    case 'v':
+    case 'h':
+    case Ctrl_H:
+    case 'l':
+    case Ctrl_L:
+    case 'H':
+    case 'L':
+    case '>':
+    case '<':
+    case '}':
+    case 'f':
+    case 'F':
+    case Ctrl_F:
+    case 'i':
+    case Ctrl_I:
+    case 'd':
+    case Ctrl_D:
+      /* window size or any count */
+      eap->addr_type = ADDR_LINES;
+      break;
+
+    case Ctrl_HAT:
+    case '^':
+      /* buffer number */
+      eap->addr_type = ADDR_BUFFERS;
+      break;
+
+    case Ctrl_Q:
+    case 'q':
+    case Ctrl_C:
+    case 'c':
+    case Ctrl_O:
+    case 'o':
+    case Ctrl_W:
+    case 'w':
+    case 'W':
+    case 'x':
+    case Ctrl_X:
+      /* window number */
+      eap->addr_type = ADDR_WINDOWS;
+      break;
+
+    case Ctrl_Z:
+    case 'z':
+    case 'P':
+    case 't':
+    case Ctrl_T:
+    case 'b':
+    case Ctrl_B:
+    case 'p':
+    case Ctrl_P:
+    case '=':
+    case CAR:
+      /* no count */
+      eap->addr_type = 0;
+      break;
+  }
+}
+
+static void cmd_with_count(char *cmd, char_u *bufp, size_t bufsize,
+                           uint32_t Prenum)
+{
+  size_t len = STRLEN(cmd);
```

Comment:
What kind of goofball lives inside a fire truck?

On April 19, 2015 5:23:51 AM EDT, Felipe Morales <notifications@github.com> wrote:
>> +    case 'b':
>> +    case Ctrl_B:
>> +    case 'p':
>> +    case Ctrl_P:
>> +    case '=':
>> +    case CAR:
>> +      /* no count */
>> +      eap->addr_type = 0;
>> +      break;
>> +  }
>> +}
>> +
>> +static void cmd_with_count(char *cmd, char_u *bufp, size_t bufsize,
>> +                           uint32_t Prenum)
>> +{
>> +  size_t len = STRLEN(cmd);
>
>I wake up at 6am on a sunday because of a fire truck siren, and sure
>enough, it was a code review from @Pyrohh ...
>
>:+1:
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/neovim/neovim/pull/2041/files#r28651168